tests/social-share-button.test.js

```
@@ -0,0 +1,568 @@
+
+    expect(instance.previousActiveElement).toBe(instance.button);
+  });
+
+  it("should move focus into the modal after opening", async () => {
+    const instance = createInstance();
+    instance.button.focus();
+
+    instance.openModal();
+
+    // The focus is moved in a setTimeout(…, 10), so we need to wait
+    await new Promise((resolve) => setTimeout(resolve, 20));
```

ESLint

[error] 'setTimeout' is not defined.

(no-undef)